source/includes/elastic-agents/_031-job-completion.md

```diff
@@ -0,0 +1,70 @@
+## Job Completion
+
+This message is a request to the plugin to terminate the agent of the job that has been completed.
```

Comment:
`The intent on this message to notify the plugin on completion of the job. The plugin may choose to terminate the elastic agent or keep it running in case if it is reusable.` ?

Reply:
I've added the note in the next section

Reply:
I mean remove the note and put in a description saying this is an optional call.